util/envDynawo.sh

```diff
@@ -739,7 +690,48 @@ build_3rd_party_version() {
       error_exit "$DYNAWO_SRC_DIR/3rdParty does not exist."
     fi
     cd $DYNAWO_SRC_DIR/3rdParty
-    bash toolchain.sh --build-type=$BUILD_TYPE_THIRD_PARTY
+    if [ $DYNAWO_BOOST_HOME_DEFAULT != true ]; then
+      BOOST_OPTION="--boost-install-dir=$DYNAWO_BOOST_HOME"
+    else
+      BOOST_OPTION=""
+    fi
+    if [ $DYNAWO_LIBARCHIVE_HOME_DEFAULT != true ]; then
+      LIBARCHIVE_OPTION="--libarchive-install-dir=$DYNAWO_LIBARCHIVE_HOME"
+    else
+      LIBARCHIVE_OPTION=""
+    fi
+    case "$DYNAWO_BUILD_TYPE_THIRD_PARTY" in
+      Debug)
+        if [ $DYNAWO_GTEST_HOME_DEFAULT != true ]; then
+          GTEST_OPTION="--gtest-install-dir=$DYNAWO_GTEST_HOME"
+        else
+          GTEST_OPTION=""
+        fi
+        ;;
+      *)
+        GTEST_OPTION=""
+        ;;
+    esac
+
+    bash toolchain.sh --build-type=$DYNAWO_BUILD_TYPE_THIRD_PARTY \
+      --sundials-install-dir=$DYNAWO_SUNDIALS_INSTALL_DIR \
```

Comment:
I didn't see the reading ot those options in toochain.sh, aren't we using global environment variable instead?

Reply:
In fact they are read